data/avro-schema/src/main/resources/avro/net/corda/data/ledger/persistence/FindTransactionIds.avsc

```diff
@@ -0,0 +1,23 @@
+{
+  "type": "record",
+  "name": "FindTransactionIds",
+  "doc": "A request to fetch transaction IDs that are in one of the given statuses.",
+  "namespace": "net.corda.data.ledger.persistence",
+  "fields": [
+  {
+    "name": "ids",
+    "type": {
+      "type" : "array",
+      "items" : "string"
+    },
+    "doc": "A list of the transaction IDs to fetch."
```

Comment:
I am not quite clear how this is used - do you give it a list of IDs and statuses, and it will check the status of the transactions and return the list filtered down to the required status?

Reply:
You give a list of transaction IDs and statuses and it will return a list of transaction IDs that are present in the database with the given statuses.